### DISCUSSION TITLE
Body framing

### DISCUSSION THREAD

**Discussion by Booleanyung** [community/community#146618] [Programming Help]
### Body

How can I make adjustments 
To booleans body frame 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hi @Booleanyung, we're happy you're here! Could you provide more details about what you’re trying to adjust regarding the "body frame" of Booleans?, giving a few more information might help someone give you a nudge in the right direction :)
